integrasjontjenester/fpsak-klient/src/main/java/no/nav/foreldrepenger/tilbakekreving/fpsak/klient/FpsakKlient.java

```diff
@@ -89,6 +91,9 @@ private Optional<SamletEksternBehandlingInfo> hentBehandlingsinfoOpt(UUID ekster
                 if (tilleggsinformasjon.contains(Tillegsinformasjon.VARSELTEKST) && lenke.getRel().equals(Tillegsinformasjon.VARSELTEKST.getFpsakRelasjonNavn())) {
                     hentVarseltekst(lenke).ifPresent(builder::setVarseltekst);
                 }
+                if (tilleggsinformasjon.contains(Tillegsinformasjon.SENDTOPPDRAG) && lenke.getRel().equals(Tillegsinformasjon.SENDTOPPDRAG.getFpsakRelasjonNavn())) {
```

Comment:
Det ser ut som vi trenger å implementere dette i k9sak+K9sakKlient for at det skal fungere for oss også, ellers vil det vel feil e i HendelseHåndtererTjeneste